### PR TITLE
Release candidate

### PR DESCRIPTION
**Pull request by ianegordon**
DO NOT MERGE
WIP

### CODE REVIEW

File: MaterialComponents.podspec [hunk 1]
```diff
@@ -196,10 +196,17 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "Cards" do |component|
+    component.ios.deployment_target = '8.0'
+    component.public_header_files = "components/#{component.base_name}/src/*.h"
+    component.source_files = "components/#{component.base_name}/src/*.{h,m}", "components/#{component.base_name}/src/private/*.{h,m}"
```
Comment (yarneo) [inline MaterialComponents.podspec]: make sure `pod lib lint` passes as /private/ is no longer required.

**Review by yarneo — APPROVED** [review]

**Comment by yarneo** [comment]
MaterialComponents passed validation.

**Comment by ianegordon** [comment]
Committer and Author CLAs have been signed.

**Comment by ianegordon** [comment]
kokoro/pr/cocoapods is failing due to implicit self casts. I expect a new compiler flag was enabled.  While these definitely should be fixed, I don't consider it to be release blocking as the casts have existed in all previous releases.

**Pull request merged**